src/main/java/Event.java

```diff
@@ -0,0 +1,51 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Defines a Event type task.
+ *
+ * @author Kai Chao
+ * @version 1.0
+ * @since 26-08-2020
+ */
+public class Event extends Task {
+
+    protected LocalDate localDate;
```

Comment:
Hi, is there any special meaning you choose protected here.

Reply:
I think I can change to private here, thanks!